app/src/main/java/com/codepath/michfeng/songswiper/models/SpotifyUser.java

```diff
@@ -1,10 +1,9 @@
 package com.codepath.michfeng.songswiper.models;
 
-public class User {
+public class SpotifyUser {
```

Comment:
not important: are your users' called SpotifyUser instead of SongSwiperUser?! :)